USAGE.md

```diff
@@ -0,0 +1,33 @@
+# How to use Testing Farm as GitHub Action in your projects
+
+Before start of using Testing Farm as GitHub Action in your projects,
+there are several steps which you have to do it.
+
+## As Testing Farm team for public or private ranch
+
+Before any implementation as Testing Farm team for API_KEY here [onboarding](https://docs.testing-farm.io/general/0.1/onboarding.html).
+
+## Setup TMT plan in GitHub / GitLab repos
+
+In your GitHub or GitLab repos create:
+* [.fmf](https://github.com/sclorg/sclorg-testing-farm/tree/main/.fmf) directory with file [version](https://github.com/sclorg/sclorg-testing-farm/blob/main/.fmf/version) which contains 1.
+* [plans](https://github.com/sclorg/sclorg-testing-farm/tree/main/plans) directory which will contain TMT plans for Testing Farm
+* create a TMT plan like it is already done here [c9s](https://github.com/sclorg/sclorg-testing-farm/blob/main/plans/c9s.fmf)
+* The TMT plan contains instructions how to setup, install and execute tests on quest provided by Testing Farm. For more information see [documentation](https://tmt.readthedocs.io/en/stable/examples.html#plans) or take an inspiration from our [plans](https://github.com/sclorg/sclorg-testing-farm/tree/main/plans) diretory
+
+## Setup GitHub Action
+
+As soon as you have your TMT plan ready then let's create an GitHub Actions for your use case
+
+### Testing by GitHub Actions
+
+Create a directory `.github/workflows` in your GitHub repository.
+
+### Testing in pull request
+
+Create a file like `testing-farm-pr.yml`
+The file can look like as is mentioned in [README.md](https://github.com/sclorg/testing-farm-as-github-action#example)
+
+### Testing when pull request is merged
```

Comment:
This one is clear, but "Testing" may lead to perception that only testing can be done, but we also can do some other production actions, like we do for sclorg containers. Maybe we can call this step as "Testing or other actions performed after a pull request is merged"?

Reply:
Fixed